src/arch/xtensa/smp/include/arch/idc.h

```diff
@@ -115,7 +115,7 @@ static void idc_irq_handler(void *arg)
 		idcietc = idc_read(IPC_IDCIETC(i), core);
 
 		if (idcietc & IPC_IDCIETC_DONE) {
-			tracev_idc("Rpy");
+			tracev_idc("idc_irq_handler(): IPC_IDCIETC_DONE");
```

Comment:
The point of this PR is to get rid of 3 letters traces everywhere without modification or adding new trace values. This can be done in another PR where necessary.